### PR TITLE
Upgrade SDK@v5.2.0

### PR DESCRIPTION
**Pull request by ndowmon**
Upgrading SDK since this work was done a little while back.

### CODE REVIEW

File: src/steps/vulnerabilities.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ function createAssetVulnerabilityEntity(
         category: 'host',
         open: assetVulnerability.status === 'vulnerable' ? true : undefined,
         severity: 'unknown',
+        numericSeverity: 5,
```
Comment (mknoedel) [inline src/steps/vulnerabilities.ts:38]: tf is dis?

**Review by mknoedel — DISMISSED** [review]
Looks good, idk what's going on with the static vulnerability of 5 though.

File: src/steps/vulnerabilities.ts (hunk 1 above)
Comment (ndowmon) [reply src/steps/vulnerabilities.ts:38]: `numericSeverity` is a required property on `Finding` entities in this version of the SDK. It ranges from 0-10 and is not optional... I went with the school of thought that we should choose the middle value. 🤷

Comment (mknoedel) [reply src/steps/vulnerabilities.ts:38]: lol, works for me

**Review by mknoedel — DISMISSED** [review]

**Review by mknoedel — APPROVED** [review]

**Review by austinkelleher — APPROVED** [review]

**Review by mknoedel — APPROVED** [review]

**Pull request merged**